demo/csv/CsvDemo.php

```
@@ -0,0 +1,141 @@
+     * - Services are configured via DI container
+     * - Multiple processing approaches can coexist
+     */
+    public function process(
+        #[InputFile(
+            allowedExtensions: ['csv'],
+            allowedTypes: ['text/csv', 'text/plain', 'application/csv'],
+            maxSize: 10 * 1024 * 1024, // 10MB
+        )]
+        FileUpload $csvFile,
+        #[Input] string $delimiter = ',',
+        #[Input] bool $hasHeader = true,
```

PHPMD (2.15.0)

Avoid unused parameters such as '$hasHeader'. (Unused Code Rules)

(UnusedFormalParameter)